### PR TITLE
Prepare 1.9.0-rc.0

### PR DESCRIPTION
**Pull request by gouthamve**
Signed-off-by: Goutham Veeramachaneni <gouthamve@gmail.com>

**What this PR does**:
Prepare 1.9.0-rc.0

- [x] `CHANGELOG.md` updated - the order of entries should be `[CHANGE]`, `[FEATURE]`, `[ENHANCEMENT]`, `[BUGFIX]`

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,9 +2,8 @@
 
 ## master / unreleased
 
-## 1.9.0 in progress
+## 1.9.0-rc.0 / 2021-04-28
 
-* [CHANGE] Fix for CVE-2021-31232: Local file disclosure vulnerability when `-experimental.alertmanager.enable-api` is used. The HTTP basic auth `password_file` can be used as an attack vector to send any file content via a webhook. The alertmanager templates can be used as an attack vector to send any file content because the alertmanager can load any text file specified in the templates list. #4129
```
Comment (pstibrany) [inline CHANGELOG.md:7]: why remove this? I think it's useful to keep it here for users to see that it's fixed on master too. WDYT?

**Review by pstibrany — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (gouthamve) [reply CHANGELOG.md:7]: Do we usually put changes that already made it into a previous release into the new release changelog?

Comment (pstibrany) [reply CHANGELOG.md:7]: You’re right, usually not.

**Review by pracucci — APPROVED** [review]
Thanks!

**Comment by bboreham** [comment]
I don't see the images for the RC - is it because tests failed here?
https://github.com/cortexproject/cortex/runs/2465206769

**Pull request merged**